README.md

```diff
@@ -57,8 +57,10 @@ This extension uses three main things in its build process:
 content scripts) into single files and manages most of the build process.
 
 1. Clone the repository.
-2. Install `node` and `yarn` (both are available through `brew`, test with node-v9.2.1 and yarn-1.3.2)
-3. On OSX, ensure command line developer tools are installed `xcode-select --install`.
+2. Install Node.js (>= v8.12.0) and Yarn (>= v1.10.0).
+
```

Comment:
According to the [nodejs.org](url) website this should be 10.16.3 LTS or 12.8.1. Or, were you using the prereq listed on the [yarn](yarnpkg.com) website. That's pretty old but whatever.

Reply:
I'm just only reiterating constraints [as currently specified](https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/master/package.json#L136-L139) in the Toolkit project

Reply:
I see no point on having them diverged. Either `engines` block in `package.json` needs to be bumped up to latest&greatest Node.js & Yarn, or README is true to constrained versions.

As you can see, I duly chose second option.